src/python/pants_release/generate_github_workflows.py

```diff
@@ -1055,17 +1055,46 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Make GitHub Release",
+                    "if": f"{IS_PANTS_OWNER} && steps.get_info.outputs.is-release == 'true'",
+                    "run": dedent(
+                        """\
+                        RELEASE_TAG=${{ steps.get_info.outputs.build-ref }}
+                        RELEASE_VERSION="${RELEASE_TAG#release_}"
+
+                        # NB: This could be a re-run of a release, in the event a job/step failed.
+                        if gh release view $RELEASE_TAG ; then
+                            exit 0
+                        fi
+
+                        GH_RELEASE_ARGS=("--notes" "")
+                        GH_RELEASE_ARGS+=("--title" "$RELEASE_TAG")
+                        if [[ $RELEASE_VERSION =~ [[:alpha:]] ]]; then
+                            GH_RELEASE_ARGS+=("--prerelease")
+                        else
+                            STABLE_RELEASE_TAGS=$(gh api -X GET -F per_page=100 /repos/{owner}/{repo}/releases --jq '.[].tag_name | sub("^release_"; "")')
```

Comment:
From [the API docs](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases), there's no suggestion to me that this only gives stable releases, and running it locally finds `2.18.0.dev4` etc. that seem to be labelled as pre-release.

One option might be something like:
```
gh release list --exclude-pre-releases --exclude-drafts --repo pantsbuild/pants | tail +1  | awk '{ print($1) }'
```
but that's not exactly nice!

Reply:
Correct that's why I filter for releases with only `[0-9.]` as the version (after the release part) in the jq filter.

Reply:
Or I do locally 😅 let me push